### ISSUE TITLE
2021.1jp に向けた speech 独自仕様の再実装

### ISSUE THREAD

**Issue by nishimotz** [nvdajp/nvdajp#292]
betajp ブランチがだんだんまともになってきた

https://ci.appveyor.com/project/TakuyaNishimoto/nvdajp/builds/39579814/artifacts

確認できていること

- ラップトップ配列 NVDA+ピリオド 1回, 2回, 3回, 4回 の各動作
- 日本語設定「フォネティック読み」
- 大文字で、ピッチ変更、ビープ、オオモジと読む、の各動作

現状の課題

- テキストカーソル移動の説明モード
- 文字の種類によるピッチ変更
- 日本語点字ビューアー
- KGS ドライバ（動作確認できていない）

**Comment by nishimotz** [comment]
音声エンジン選択で下記のエラーが出ている

```
ERROR - synthDriverHandler.getSynthList (21:30:18.373) - MainThread (6056):
Error while importing SynthDriver espeak
Traceback (most recent call last):
  File "synthDriverHandler.pyc", line 397, in getSynthList
  File "synthDriverHandler.pyc", line 386, in _getSynthDriver
  File "importlib\__init__.pyc", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 668, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 638, in _load_backward_compatible
  File "synthDrivers\espeak.pyc", line 17, in <module>
ImportError: cannot import name 'BooleanDriverSetting' from 'driverHandler' (D:\work\nvda\nvda-betajp-portable\library.zip\driverHandler.pyc)
INFO - synthDriverHandler.setSynth (21:30:20.841) - MainThread (6056):
Loaded synthDriver nvdajp_jtalk
```

**Comment by nishimotz** [comment]
EnhancedPhoneticReading アドオンのテクニックがよいように思えてきた。

具体的には、外部モジュールから speech.speakTextInfo などにモンキーパッチを当てるやり方である。

**Comment by nishimotz** [comment]
当初の方針で実装をやりきって、だいたいバグが取れたので、日本語アルファ版を更新しました。
